test/services/account/services/accounts.test.js

```diff
@@ -109,10 +109,10 @@ describe('Account Service', () => {
 			};
 
 			await new Promise((resolve, reject) => {
-				accountService.create(newAccount)
+				accountService
+					.create(newAccount)
 					.then(() => {
-						reject(new Error('This call should fail because '
-                        + 'of an already existing user with the same username'));
+						reject(new Error('This call should fail because ' + 'of an already existing user with the same username'));
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6074601)